rllib/env/policy_server_input.py

```diff
@@ -64,7 +65,7 @@ class PolicyServerInput(ThreadingMixIn, HTTPServer, InputReader):
     """
 
     @PublicAPI
-    def __init__(self, ioctx, address, port, idle_timeout=3.0):
+    def __init__(self, ioctx, address, port, idle_timeout=3.0, use_json=False):
```

Comment:
you've added this as an option, but is this used at any point now in the tests/examples?

Reply:
Good catch, will remove.
Leftover from my other work, which made me discover this bug in the first place :)